distributed/core.py

```diff
@@ -673,6 +673,12 @@ def stop(self):
             future = listener.stop()
             if inspect.isawaitable(future):
                 _stops.add(future)
+            try:
+                abort_handshaking_comms = listener.abort_handshaking_comms
+            except AttributeError:
+                pass
```

Comment:
I suspect this is merely here to maintain backwards compat in case a user implemented their own thing?